src/components/FeatureViews/AdminPortal/UnpaidSessions.js

```diff
@@ -12,23 +15,39 @@ import "./AdminPortal.scss";
 
 const UnpaidSessions = () => {
 
-	const UnpaidList = useSelector(({Admin}) => Admin.Unpaid) || [];
-	const studentList = useMemo(() => UnpaidList.map(({student}) => student), [
-		UnpaidList,
-	]);
-	const courseList = useMemo(() => UnpaidList.map(({course}) => course), [
-		UnpaidList,
-	]);
+	const QUERIES = {
```

Comment:
You don't need this - Neil only did this in the accounts tab because he had a lot of queries to pull from. You only need to create an UNPAID_SESSIONS_QUERY const and set that equal to what you have starting from line 19 - 39